src/status_im/chat/handlers.cljs

```diff
@@ -291,18 +289,18 @@
 (register-handler :add-chat-loaded-callback
   (fn [db [_ chat-id callback]]
     (log/debug "Add chat loaded callback: " chat-id callback)
-    (update-in db [::chat-loaded-callbacks chat-id] conj callback)))
+    (update-in db [:chat-loaded-callbacks chat-id] conj callback)))
```

Comment:
ns qualified keywords are fine when we are going to use them in single ns, so that we can keep keywords simple and have a guaranty that keywords will not intersect with anything else in db. But `::chat-loaded-callbacks` is not the best example, as `status-im.chat.handlers` itself, it's too big...